rclcpp/src/rclcpp/experimental/executors/events_executor/events_executor.cpp

```diff
@@ -62,7 +62,6 @@ EventsExecutor::EventsExecutor(
       //    ---> we need to wake up the executor so that it can terminate
       // - a node or callback group guard condition is triggered:
       //    ---> the entities collection is changed, we need to update callbacks
-      notify_waitable_event_pushed_ = false;
```

Comment:
this is now set to false inside `refresh_current_collection_from_callback_groups`